tests/fixtures/starknet.py

```diff
@@ -33,6 +34,10 @@
 
 @pytest_asyncio.fixture(scope="session")
 async def starknet(worker_id, request, blockhashes) -> AsyncGenerator[Starknet, None]:
+    # apply cairo-rs-py monkey patch to use the rust VM
+    cairo_rs_py_monkeypatch()
```

Comment:
Currently failling

```13 failed, 24 passed, 2 skipped, 1 xfailed in 177.30s (0:02:57)```

Reply:
tests are failing on the `unsigned_div_rem` bugs of the cairo vm solved few months ago; the embedded version in cairo-rs-py needs to be updated

Reply:
Gotcha 🤙 should I make an issue to fork/pr on cairo-rs-py to update this?